canopy_waf_mod.F90

```diff
@@ -112,21 +118,21 @@ SUBROUTINE CANOPY_WAF( HCM, LAMDARS, RSL_OPT, HREF, FLAMEH, FIRETYPE, &
             waf = 1.0
         else
 
-            if (RSL_OPT .eq. 1) then  !set lamda_rs = 1 to avoid double counting RSL effects
-                lamda_rs =  1.0
-            else                      !set to lamda_rs to namelist input LAMDARS
-                lamda_rs = LAMDARS
-            end if
+!            if (RSL_OPT .eq. 1) then  !set lambda_rs = 1 to avoid double counting RSL effects
+!                lambda_rs =  1.0
+!            else                      !set to lambda_rs to namelist input LAMBDARS
+            lambda_rs = LAMBDARS
+!            end if
```

Comment:
.

Reply:
@zmoon Yes, I agree with this and others above.  I have removed those commented out lines.

Reply:
swell